x-pack/test/security_api_integration/tests/anonymous/login.ts

```diff
@@ -118,7 +118,7 @@ export default function ({ getService }: FtrProviderContext) {
 
         expect(unauthenticatedResponse.headers['set-cookie']).to.be(undefined);
         expect(unauthenticatedResponse.headers['content-security-policy']).to.be(
-          `script-src 'unsafe-eval' 'self'; worker-src blob: 'self'; style-src 'unsafe-inline' 'self'`
+          `script-src 'self'; worker-src blob: 'self'; style-src 'unsafe-inline' 'self'`
         );
```

Comment:
You can also loosen the assertion for this and the other `security_api_integration` tests:

```diff
diff --git a/x-pack/test/security_api_integration/tests/anonymous/login.ts b/x-pack/test/security_api_integration/tests/anonymous/login.ts
index 63b363c2259..bdb504014f9 100644
--- a/x-pack/test/security_api_integration/tests/anonymous/login.ts
+++ b/x-pack/test/security_api_integration/tests/anonymous/login.ts
@@ -117,9 +117,7 @@ export default function ({ getService }: FtrProviderContext) {
           .expect(401);
 
         expect(unauthenticatedResponse.headers['set-cookie']).to.be(undefined);
-        expect(unauthenticatedResponse.headers['content-security-policy']).to.be(
-          `script-src 'self'; worker-src blob: 'self'; style-src 'unsafe-inline' 'self'`
-        );
+        expect(unauthenticatedResponse.headers['content-security-policy']).to.be.a('string');
         expect(unauthenticatedResponse.text).to.contain('We couldn&#x27;t log you in');
       });
     });
```

Reply:
I assume you mean because it's not important for these test what the content of this header is, just that it's set? I'll do that